tools/markdown-cms/local.ts

```diff
@@ -1,21 +1,23 @@
-import { ICms } from "df/tools/markdown-cms";
-import { Tree } from "df/tools/markdown-cms/tree";
+import { ICms } from "@dataform-tools/markdown-cms";
+import { Tree } from "@dataform-tools/markdown-cms/tree";
 import * as fs from "fs";
-import { basename, dirname, join } from "path";
+import { join } from "path";
 import { promisify } from "util";
 
-export class LocalCms implements ICms {
+export class LocalCms<T> implements ICms<T> {
   constructor(private rootPath: string) {}
 
   public async get(path = "") {
     const isDir = await this.isDirectory(path);
     const actualFilePath = isDir
       ? join(this.rootPath, path, "index.md")
       : join(this.rootPath, path);
-    const rawContent = await promisify(fs.readFile)(actualFilePath, "utf8");
+    const rawContent = (await promisify(fs.exists)(actualFilePath))
+      ? await promisify(fs.readFile)(actualFilePath, "utf8")
+      : "";
```

Comment:
```suggestion
    const rawContent = fs.existsSync(actualFilePath)
      ? fs.readFileSync(actualFilePath, "utf8")
      : "";
```

All file `promisify` stuff can me removed using `fs-extra`; looks neater but not really required

Reply:
Nice stuff!

Reply:
Ack - don't want to add this dep yet as we don't have it in open-source, although I agree it's nicer!